pkg/pdapi/pd_control.go

```diff
@@ -33,6 +33,8 @@ type PDControlInterface interface {
 	GetPDClient(namespace Namespace, tcName string, tlsEnabled bool) PDClient
 	// GetClusterRefPDClient provides PDClient of the tidb cluster.
 	GetClusterRefPDClient(namespace Namespace, tcName string, clusterDomain string, tlsEnabled bool) PDClient
+	// GetPeerPDClient provides PD Client of the tidb cluster from peerURL.
+	GetPeerPDClient(namespace Namespace, tcName string, tlsEnabled bool, peerURL string, peerKey string) PDClient
```

Comment:
`peerURL` often refers to `xxx:2380` (not `xxx:2379`) in PD/etcd, but it's peer-member's client URL here, how about giving it a better name?

Reply:
It's a little tricky in multi-cluster. If the URL contains the pod name, we can ignore the port spec in the peer service.
It's the opposite concept to ClusterIP in the concept of the VPC-native Kubernetes cluster. We need the FQDN and the pod IP available in the VPC scope rather than the ClusterIP. We can refer to https://cloud.google.com/kubernetes-engine/docs/how-to/alias-ips for more details about the VPC-native cluster. In TiDB-Operator, the pod IP is provided by the peer service.